### ISSUE TITLE
Add example for hybrid local/remote development config

### ISSUE THREAD

**Issue by landism** [tilt-dev/tilt#4933]
## Describe the Feature You Want
Tilt's [Local vs Remote Services](https://docs.tilt.dev/local_vs_remote.html#hybrid) doc says hybrid local/remote development is the most common setup we see. An example project demonstrating this setup could be useful.

**Comment by arkus7** [comment]
Is there any update on this? I went through the whole documentation on the [docs.tilt.dev](https://docs.tilt.dev) website, but I haven't found anything regarding how to set this up.

**Comment by VanTigranyan** [comment]
> 

I didn't find anything outside of official docs either. I assume since there are examples of both remote and local services we should basically combine them making the setup hybrid.

**Comment by ekovacs** [comment]
yes, it would be great to have more resource/guides to set up hybrid: run most stuff in local tilt, but use 1 or 2 services from a remote cluster.
